### PR TITLE
screendesign_20626_wireframe_design

### PR DESCRIPTION
**Pull request by truong-vu-mulodo**
- Create wireframe for following screens:
  - [x] Login/Logout + Home
  - [x] Register User Info
  - [x] Update User Info 
  - [x] Change Password  
  - [x] Search User by username/firstname/lastname  
  - [x] List all posts
  - [x] List all user posts (including current user)
  - [x] Create/Edit/Delete a post
  - [x] Create/Edit/Delete comments for a post

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-projectname-lastname.firstname
+miniblog-nguyen.thuc
 ==============================
 
-Template repo for Mulodo training. (ex: miniblog-truong.vu) 
+Make a change for creating pull request.
```
Comment (truong-vu-mulodo) [inline README.md:4]: Code revew example.

**Comment by nguyen-thuc-mulodo** [comment]
•Create wireframe for following screens:
- [x] Login/Logout + Home

◦[wip] Register User Info
- [ ] Update User Info 
- [ ] Change Password
- [ ] Search User by username/firstname/lastname
- [ ] List all posts
- [ ] List all user posts (including current user)
- [ ] Create/Edit/Delete a post
- [ ] Create/Edit/Delete comments for a post

**Comment by truong-vu-mulodo** [comment]
@nguyen-thuc-mulodo: There is no preview before save post function?

**Comment by nguyen-thuc-mulodo** [comment]
Yes, there isn't preview. Now I will add preview page into my wireframe

**Comment by truong-vu-mulodo** [comment]
@nguyen-thuc-mulodo: Looks good! Just merge into master and go ahead.

**Pull request merged**